eng/pipelines/extra-platforms/runtime-extra-platforms-wasm.yml

```diff
@@ -322,3 +322,29 @@ jobs:
       isWasmOnlyBuild: ${{ parameters.isWasmOnlyBuild }}
       scenarios:
         - normal
+
+  # Library tests with full threading & blocking .Wait
+  - template: /eng/pipelines/common/templates/wasm-library-tests.yml
+    parameters:
+      platforms:
+        - browser_wasm
+      nameSuffix: _Threading_BlockingWait
+      extraBuildArgs: /p:WasmEnableThreads=true /p:_WasmAllowBlockingWait:true /p:AotHostArchitecture=x64 /p:AotHostOS=$(_hostedOS)
+      alwaysRun: true
```

Comment:
@akoeplinger i'm not sure what should be here. I want this to run when `/azp run runtime-wasm`